docs/guide/known-limitations.md

```diff
@@ -72,11 +72,7 @@ To remove the differences, you can create custom implementations of those functi
 | DAYS          | =DAYS(-1, 0)                                               |          NUM |      -1,0000 |        NUM |
 | DAYS          | =DAYS(0, -1)                                               |          NUM |       1,0000 |        NUM |
 | DATEDIF       | =DATEDIF(-1, 0, "Y")                                       |          NUM |       0,0000 |        NUM |
-| RATE          | =RATE(12, -100, 400, 100, 1)                               |      -0,4997 |       0,3100 |    -0,4997 |
-| RATE          | =RATE(12, -100, 400, 1, 1)                                 |      -0,9901 |       0,3200 |    -0,9901 |
-| RATE          | =RATE(12, -100, 400, 0, 1)                                 |          NUM |       0,3200 |        NUM |
-| RATE          | =RATE(12, -100, 400, -100, 1)                              |          NUM |       0,3200 |        NUM |
-| RATE          | =RATE(0.9, -100, 400)                                      |      -0,7962 |          NUM |    -0,7962 |
+| RATE          | =RATE(12, -100, 400, 0, 1)                                 |      -1,0000 |          NUM |        NUM |
```

Comment:
It is weird that on this docs page we use the comma as the decimal separator. The ODFF standard uses the dot. Maybe make a new issue about that?

Reply:
Most (all?) english speaking cultures use `.` so I believe we also should. Issue: https://github.com/handsontable/hyperformula/issues/942